pkg/controller/node/node_controller.go

```diff
@@ -360,6 +378,42 @@ func (ctrl *Controller) makeMasterNodeSchedulable(node *corev1.Node) error {
 	return nil
 }
 
+func (ctrl *Controller) addMachineOSBuild(obj interface{}) {
+	curMOSB := obj.(*mcfgv1alpha1.MachineOSBuild)
+
+	config, _ := ctrl.client.MachineconfigurationV1alpha1().MachineOSConfigs().Get(context.TODO(), curMOSB.Spec.MachineOSConfig.Name, metav1.GetOptions{})
+
+	mcp, _ := ctrl.mcpLister.Get(config.Spec.MachineConfigPool.Name)
+	ctrl.enqueueMachineConfigPool(mcp)
+}
+
+func (ctrl *Controller) updateMachineOSBuild(old, cur interface{}) {
+	curMOSB := cur.(*mcfgv1alpha1.MachineOSBuild)
+
+	config, _ := ctrl.client.MachineconfigurationV1alpha1().MachineOSConfigs().Get(context.TODO(), curMOSB.Spec.MachineOSConfig.Name, metav1.GetOptions{})
+
+	mcp, _ := ctrl.mcpLister.Get(config.Spec.MachineConfigPool.Name)
+	ctrl.enqueueMachineConfigPool(mcp)
+}
+
+func (ctrl *Controller) deleteMachineOSBuild(obj interface{}) {
+	pool, ok := obj.(*mcfgv1alpha1.MachineOSBuild)
+	if !ok {
+		tombstone, ok := obj.(cache.DeletedFinalStateUnknown)
+		if !ok {
+			utilruntime.HandleError(fmt.Errorf("couldn't get object from tombstone %#v", obj))
+			return
+		}
+		pool, ok = tombstone.Obj.(*mcfgv1alpha1.MachineOSBuild)
+		if !ok {
+			utilruntime.HandleError(fmt.Errorf("tombstone contained object that is not a MOSB %#v", obj))
+			return
+		}
+	}
+	klog.V(4).Infof("Deleting MachineConfigPool %s", pool.Name)
+	// TODO(abhinavdahiya): handle deletes.
```

Comment:
this looks like got carried over from somewhere else

Reply:
yep, that was my fault sorry!